src/main/java/dev/jlibra/admissioncontrol/AdmissionControl.java

```diff
@@ -0,0 +1,149 @@
+package dev.jlibra.admissioncontrol;
+
+import static java.util.stream.Collectors.toList;
+
+import java.io.IOException;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.google.protobuf.ByteString;
+
+import admission_control.AdmissionControlGrpc;
+import admission_control.AdmissionControlGrpc.AdmissionControlBlockingStub;
+import admission_control.AdmissionControlOuterClass.SubmitTransactionRequest;
+import admission_control.AdmissionControlOuterClass.SubmitTransactionResponse;
+import dev.jlibra.AccountState;
+import dev.jlibra.KeyUtils;
+import dev.jlibra.LibraHelper;
+import dev.jlibra.admissioncontrol.query.GetAccountState;
+import dev.jlibra.admissioncontrol.query.UpdateToLatestLedgerResult;
+import dev.jlibra.admissioncontrol.transaction.SubmitTransactionResult;
+import dev.jlibra.admissioncontrol.transaction.Transaction;
+import dev.jlibra.admissioncontrol.transaction.TransactionArgument.Type;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import types.GetWithProof.GetAccountStateRequest;
+import types.GetWithProof.RequestItem;
+import types.GetWithProof.UpdateToLatestLedgerRequest;
+import types.GetWithProof.UpdateToLatestLedgerResponse;
+import types.Transaction.Program;
+import types.Transaction.RawTransaction;
+import types.Transaction.SignedTransaction;
+import types.Transaction.TransactionArgument;
+import types.Transaction.TransactionArgument.ArgType;
+
+public class AdmissionControl {
+
+    private String host;
+
+    private int port;
+
+    public AdmissionControl(String host, int port) {
+        this.host = host;
+        this.port = port;
+    }
+
+    private static Map<Type, ArgType> jlibraArgumentTypeToGrpcArgumentType;
+
+    static {
+        jlibraArgumentTypeToGrpcArgumentType = new HashMap<>();
+        jlibraArgumentTypeToGrpcArgumentType.put(Type.ADDRESS, ArgType.ADDRESS);
+        jlibraArgumentTypeToGrpcArgumentType.put(Type.U64, ArgType.U64);
+    }
```

Comment:
I'd move statics to the top

Reply:
I'm working on another example in another branch where this is fixed now (get_transactions_example).